src/lib/stitching/gravity/stitching.ts

```diff
@@ -74,6 +81,32 @@ export const gravityStitchingEnvironment = (
             })
           },
         },
+        formattedEndAt: {
+          fragment: gql`
+            ... on ViewingRoom {
+              startAt
+              endAt
+            }
+          `,
+          resolve: ({ startAt: _startAt, endAt: _endAt }) => {
+            const startAt = moment(_startAt)
+            const endAt = moment(_endAt)
+
+            if (endAt > moment().add(30, "days")) {
+              return null
+            }
+            if (endAt < startAt) {
+              return "Closed"
+            }
+
+            const label =
+              "Closes in " +
+              (moment.duration(
+                endAt.diff(startAt)
+              ) as DistancePlugin).distance()
```

Comment:
Also curious if moment [Time to X](https://momentjs.com/docs/#/displaying/to/) can accomplish similar thing or maybe it's not that close to design?

Reply:
The distance plugin covers many more use-cases and models itself after rails. I can copy the source into MP if the dep is a concern. It's not too big. 

I'm not the greatest with dates and happy to pass this off, otherwise we can iterate.

Reply:
I'm not too concerned about the dep as long as it's well-tested and served our needs! 👍